src/components/login.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import './styling.css';
+
+const style = {
```

Comment:
Always put your styles in a CSS file

Reply:
Okay, this has been noted.Thanks for the feedback